doc/releases/migration-guide-3.7.rst

```diff
@@ -714,6 +687,33 @@ Watchdog
 Bluetooth
 *********
 
+Bluetooth HCI
+=============
+
+ * A new HCI driver API was introduced (:github:`72323`) and the old one deprecated. The new API
+   follows the normal Zephyr driver model, with devicetree nodes, etc. The host now
+   selects which driver instance to use as the controller by looking for a ``zephyr,bt-hci``
+   chosen property. The devicetree bindings for all HCI drivers derive from a common
+   ``bt-hci.yaml`` base binding.
```

Comment:
wouldnt it be better to a link to the `dtcompatible`?

Reply:
If you're referring to the base binding, then that doesn't have its own compatible string

Reply:
silly me :)